lld/test/ELF/riscv-tlsdesc.s

```diff
@@ -30,10 +30,10 @@
 # RUN: llvm-objdump --no-show-raw-insn -M no-aliases -h -d a.32.ie | FileCheck %s --check-prefix=IE32
 
 # RUN: llvm-mc -triple=riscv64 -filetype=obj d.s -o d.64.o
-# RUN: not ld.lld -shared -soname=d.64.so -o d.64.so d.64.o 2>&1 | FileCheck %s --check-prefix=BADTLSLABEL
+# RUN: ld.lld -shared -soname=d.64.so -o d.64.so d.64.o 2>&1 | FileCheck %s --check-prefix=BADTLSLABEL --allow-empty
```

Comment:
--allow-empty is an anti-pattern. Avoid.

Replace `2>&1 | ...` with --fatal-warnings

Reply:
Oh, I wasn't aware. Thanks. Do you think we should update the FileCheck.rst to note that other options/patterns are preferred ... or maybe in the TestingGuide.rst?